eng/pipelines/tagger.yml

```diff
@@ -93,7 +64,26 @@ steps:
     Set-Location '$(Pipeline.Workspace)/VS'
     git init
     git remote add origin https://$(System.AccessToken)@dev.azure.com/devdiv/DevDiv/_git/VS
-    git fetch --depth 1 origin $(VsCommitId)
-  displayName: Fetch VS Repo
-- powershell: . '$(Build.SourcesDirectory)/eng/scripts/CreateTagFromVSCommitTitle.ps1' -vsDirectory '$(Pipeline.Workspace)/VS/' -vsCommitId '$(VsCommitId)'
-  displayName: Create VS Insertion Tag
\ No newline at end of file
+    git fetch --depth 100 origin $(Build.SourceVersion)
+    git checkout FETCH_HEAD
```

Comment:
Does this need to actually check out a working directory? It could be that having a bare clone with no working directory is faster.

Reply:
I can integrate the version in git log. Will update the PR.